bin/vitamin.js

```diff
@@ -299,15 +297,15 @@ program
             if (!code) {
                 return;
             }
-            console.error(chalk.red(
-                `${chalk.bold('\n\nServer process exited unexpectedly. \n')}` +
-                '- If it is an `EADDRINUSE error, you might want to change the `server.port` key' +
+            logger.error('Server process exited unexpectedly');
+            logger.info(
+                '- If it is an `EADDRINUSE` error, you might want to change the `server.port` key' +
```

Comment:
we should detect if it is a `EADDRINUSE` instead of asking the user to check if it is...